src/sdk/namespace_fs.js

```diff
@@ -180,15 +180,17 @@ class NamespaceFS {
      *  bucket_id: string;
      *  namespace_resource_id?: string;
      *  access_mode: string;
+     *  versioning: boolean;
```

Comment:
Doesn't is suppose to string? one of those: 'DISABLED', 'SUSPENDED', 'ENABLED', maybe you should add versioning enum to nb.d.ts?

Reply:
it's a leftover, replaced it with the correct value